wazo_calld_client/commands/switchboards.py

```diff
@@ -18,9 +18,10 @@ def list_queued_calls(self, switchboard_uuid):
 
         return r.json()
 
-    def answer_queued_call_from_user(self, switchboard_uuid, call_id):
+    def answer_queued_call_from_user(self, switchboard_uuid, call_id, line_id=None):
         url = self._client.url(self.resource, switchboard_uuid, 'calls', 'queued', call_id, 'answer')
-        r = self.session.put(url)
+        payload = {'line_id': line_id} if line_id else None
+        r = self.session.put(url, json=payload)
```

Comment:
The tests in `test_switchboards.py` must also be modified to reflect these changes.

Reply:
done

Reply:
Since we have tests here I have added zuul/black too.